web-client/src/components/validation/Validation.vue

```diff
@@ -24,60 +24,122 @@
       </div>
     </div>
     <div v-show="validationStore.step === 0">
-      <div
-        v-if="memoryStore.models.size > 0"
-        class="grid gris-cols-1 md:grid-cols-2 lg:grid-cols-3 items-stretch gap-8 mt-8"
-      >
-        <div
-          v-for="[path, metadata] in memoryStore.models"
-          :key="path"
-          class="contents"
-        >
-          <ButtonCard
-            :click="() => selectModel(path)"
-            :button-placement="'left'"
-          >
+      <div class="flex flex-col gap-16">
+        <div v-if="memoryStore.models.size > 0">
+          <IconCard>
             <template #title>
-              {{ taskTitle(metadata.taskID) }}
+              Model Library — <span class="italic">Downloaded and Ready to Test</span>
             </template>
-            <template #text>
-              <div class="grid grid-cols-2 justify-items-between">
-                <p class="contents">
-                  <span>Model:</span>
-                  <span>{{ metadata.name.substring(0, 20) }}</span>
-                </p>
-                <p class="contents">
-                  <span>Date:</span>
-                  <span>{{ metadata.date }} at {{ metadata.hours }}</span>
-                </p>
-                <p class="contents">
-                  <span>Size:</span><span>{{ metadata.fileSize }} kB</span>
-                </p>
+            <template #content>
+              Test any model below against a validation dataset.
+              The models listed were downloaded from the remote server.
```

Comment:
clarify that this is only for federated tasks, for decentralized this is not possible.

BTW for the future, do we also want to offer sharing of models via providing an URL in general (i can email it to you e..g.), even outside FL?

Reply:
> clarify that this is only for federated tasks, for decentralized this is not possible.

I changed it so it only displays models for federated tasks 👍 

> BTW for the future, do we also want to offer sharing of models via providing an URL in general (i can email it to you e..g.), even outside FL?

do you mean for the initial model, on task creation?

Reply:
future model sharing: can be at any stage of training. the interface is always my local model library. i'll save some good ones when accuracy is good, and later i might want to share some of these checkpoints with you, for your purposes of doing inference for example, or to continue training/finetuning from my checkpoint instead of from the initial model of the task. the library already supports saving to local disk, but not import yet (except what you did now with FL server)

anyway this is not time sensitive